### PR TITLE
Update shell placeholder screen with new designs

### PR DESCRIPTION
**Pull request by Parveshdhull**
### Summary

- PR adds empty state cards and updates shell screen typography as per new changes
- Moves placeholder styles to style namespace
- Renames `constants` to `shell.constants`, to avoid confusion

status: ready

### CODE REVIEW

File: src/react_native/core.cljs [hunk 1]
```diff
@@ -4,7 +4,8 @@
             ["@react-native-community/blur" :as blur]
             [react-native.flat-list :as flat-list]
             [react-native.section-list :as section-list]
-            [react-native.platform :as platform]))
+            [react-native.platform :as platform]
+            ["react-native-linear-gradient" :default LinearGradient]))
```
Comment (flexsurfer) [inline src/react_native/core.cljs]: see `react-native.linear-gradient`

File: src/status_im2/contexts/shell/style.cljs [hunk 2]
```diff
@@ -25,11 +25,48 @@
...
+   :weight :regular
+   :align  :center
+   :style  {:margin-top 4
+            :color      colors/white}})
+
+;; Shell
+(defn jump-to-text [status-bar-height]
+  {:size   :heading-1
+   :weight :semi-bold
+   :style  {:color         colors/white
```
Comment (flexsurfer) [inline src/status_im2/contexts/shell/style.cljs]: better to have only style in this ns, and do not mix with props, because it will be confusing

Comment (flexsurfer) [reply src/status_im2/contexts/shell/style.cljs]: probably we should add this to guidelines cc @ilmotta

File: src/status_im2/contexts/shell/style.cljs [hunk 3]
```diff
@@ -25,11 +25,46 @@
...
+   :align-items         :center
+   :accessibility-label :shell-placeholder-view})
+
+(def placeholder-image
+  {:margin-top 186
+   :width      120
+   :height     120})
+
+(def placeholder-title
+  {:size   :paragraph-1
```
Comment (flexsurfer) [inline src/status_im2/contexts/shell/style.cljs]: here

File: src/status_im2/contexts/shell/style.cljs [hunk 4]
```diff
@@ -25,11 +25,46 @@
...
+   :weight :semi-bold
+   :style  {:margin-top 20
+            :color      colors/white}})
+
+(def placeholder-subtitle
+  {:size   :paragraph-2
```
Comment (flexsurfer) [inline src/status_im2/contexts/shell/style.cljs]: and here as well

**Review by flexsurfer — APPROVED** [review]

**Comment by Parveshdhull** [comment]
Hi @flexsurfer, Thank you for reviewing PR.
Should I move these too? (old code)
https://github.com/status-im/status-mobile/blob/cbb65719544f938f017974f9c96db4a06c3263af/src/status_im2/contexts/shell/cards/style.cljs#L33

**Comment by flexsurfer** [comment]
> Hi @flexsurfer, Thank you for reviewing PR. Should I move these too? (old code)
> 
> https://github.com/status-im/status-mobile/blob/cbb65719544f938f017974f9c96db4a06c3263af/src/status_im2/contexts/shell/cards/style.cljs#L33

yes please, all in this file

**Pull request merged**